docs/course.md

```diff
@@ -254,6 +270,9 @@ Modules are optional and do not affect any behavior by default. The order in whi
 The above configuration can result in the following view for students:
 
 ![Assessments grouped by module on the student assessments overview.](assessment-units.png)
+/// caption
+Assessments grouped by module on the student assessments overview.
```

Comment:
captioning images syntax

Reply:
This is more of just an example, and in this case, it doesn't hurt readability.

Reply:
It's helpful to see what Material can do this, but I'd lean towards removing this for now until we figure out a broader strategy (I lean towards generally not having captions).